code/game/objects/items/nuke_tools.dm

```diff
@@ -0,0 +1,52 @@
+//Items for nuke theft traitor objective
+
+//the nuke core - objective item
+/obj/item/nuke_core
+	name = "plutonium core"
+	desc = "Extremely radioactive. Wear goggles."
+	icon = 'icons/obj/nuke_tools.dmi'
+	icon_state = "plutonium_core"
+	var/pulse = 0
+
+/obj/item/nuke_core/process()
+	if(!pulse)
+		pulse = 1
+		for(var/mob/living/L in range(8,loc))
+			L.irradiate(50)
+		for(var/mob/living/L in range(3,loc))
```

Comment:
Not every process call and it's to irradiate people who are closer more than people who are farther away. If you're close enough, you get irradiated twice. That is the intention.